library/src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -18,9 +18,13 @@
 import POGOProtos.Networking.Envelopes.RequestEnvelopeOuterClass.RequestEnvelope.AuthInfo;
 import POGOProtos.Networking.Envelopes.SignatureOuterClass;
 import POGOProtos.Networking.Envelopes.Unknown6OuterClass;
+import POGOProtos.Networking.Requests.RequestTypeOuterClass.RequestType;
```

Comment:
you are right, is now needed in the other class: PlayerProfile. Can be removed from there!